src/pages/patterns/filtering/index.mdx

```diff
@@ -69,6 +69,62 @@ Multiple category selection is most commonly manifested in either a vertical lis
 </Column>
 </Row>
 
+### Advanced filtering
+
+_Advanced search_ can be used when the user requires to search and filter a list of items by choosing more than one filter from multiple filter categories. This pattern is especially useful when the number of items within each category is dynamic, such as filtering a table by a user or user defined tags.
+
+Advanced search is also commonly used when there is not the requried amount of space on the screen to accommodate a vertical list.
```

Comment:
replace:
"Filtered search is also recommended when there is not enough space to accommodate a vertical list of filters."